src/modules/adaptors/Frax/CollateralFTokenAdaptorV1.sol

```diff
@@ -0,0 +1,86 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.21;
+
+import { IFToken } from "src/modules/adaptors/Frax/FTokenAdaptor.sol";
+import { CollateralFTokenAdaptorV2 } from "src/modules/adaptors/Frax/CollateralFTokenAdaptorV2.sol";
+
+interface V1FToken {
```

Comment:
could move this into a separate file, but imo we need to do refactoring anyways, so this isn't really a big deal to leave it here, so could also just add a natspec explaining why we have 2 different interfaces

Reply:
Added a natspec description, though wouldn't mind your quick review of it.

Reply:
Remove title and author then this natspec is good.

Reply:
Done